### DISCUSSION TITLE
How to debugging Typescript compiled with Parcel

### DISCUSSION THREAD

**Discussion by sebmedia** [gatsbyjs/gatsby#36175] [Help]
As announced with release [v4.9](https://www.gatsbyjs.com/docs/reference/release-notes/v4.9/) Gatsby supports now Typescript for `gatsby-node.ts` and `gatsby-config.ts`. 🚀  

The [official documentation "Debugging the Build Process"](https://www.gatsbyjs.com/docs/debugging-the-build-process/) describes how to debugging Javascript. Currently I can not find a way to debug the code of the `gatsby-node.ts` file. 

What are the steps to debug the `gatsby-node.ts` file?

**Comment by letzdoo-js** [comment]
Same question for me.

I suppose best way for now is to move back to .js ;-)
